src/service/user-service.js

```diff
@@ -0,0 +1,83 @@
+const UserModel = require("../models/user-model");
```

Comment:
somewhere a model with a capital letter, somewhere with a small one. Fix for one style